main_GANs.py

```diff
@@ -10,21 +10,30 @@
   main_GANs.py --version
   main_GANs.py [--epochs=100] [--optimizer='CGD'] [--learning_rate=0.0005]
   main_GANs.py [-e 100] [-o 'CGD'] [-l 0.0005]
+  main_GANs.py [--model='MLP']
+  main_GANs.py [-m 'CNN'] [--display] [--save]
+
   
 Options:
   -h, --help                  Show this screen.
   --version                   Show version.
+  --display                   Use matplotlib to plot results.
+  --save                      Save a model.
   -e, --epochs=<n>            Number of epochs [default: 100].
   -o, --optimizer=<str>       Optimizer name [default: Jacobi].
   -l, --learning_rate=<f>     Learning rate [default: 0.01].
+  -m, --model=<str>           Implementation of GANs model. Multi-layer 
+                              perceptrons NN (MLP), convolutional NN (CNN) 
+                              [default: MLP].
 """
-
+import sys
 from docopt import docopt
 from MLP_GANs_object import *
 from DCGANs_object import *
```

Comment:
I noticed that `matplotlib` is not included here, but is coming from a different place. Not necessarily need to clean up right now, but should be fixed at some point.

Reply:
You are right, it is imported in GANs_abstract_object.py to save images. But it is good to include it in main as well.